app/components/curriculum-inventory/sequence-block-overview.hbs

```diff
@@ -299,6 +299,58 @@
               {{/if}}
             </span>
           </div>
+          <div class="block starting-academic-level" data-test-starting-academic-level>
+            <label for="academic-level-{{templateId}}">{{t "general.startLevel"}}:</label>
+            {{#if @canUpdate}}
+              <EditableField
+                @value={{this.startingAcademicLevel.name}}
+                @save={{perform this.changeStartingAcademicLevel}}
+                @close={{this.revertStartingAcademicLevelChanges}}
+              >
+                <select
+                  id="starting-academic-level-{{templateId}}"
+                  {{on "change" this.setStartingAcademicLevel}}
+                >
+                  {{#each (sort-by "level" this.academicLevels) as |obj|}}
+                    <option value={{obj.id}} selected={{eq obj.id this.startingAcademicLevel.id}}>{{obj.name}}</option>
+                  {{/each}}
+                </select>
+                {{#each (await (compute this.getErrorsFor "startingAcademicLevel")) as |message|}}
```

Comment:
Can be replaced with `<ValidationError @errors={{get-errors-for this.startingAcademicLevel}} />``

Reply:
:heavy_check_mark: